### DISCUSSION TITLE
Globals history/context

### DISCUSSION THREAD

**Discussion by leowebguy** [craftcms/cms#14151] [Ideas] [answered]
Hey guys,
Same as entries, globals can be mistakenly overridden and saved.
Would be possible to add history/context to globals?

**Comment by brandonkelly** [comment, marked as answer]
We’re [in the process](https://www.craftcms.com/blog/entrification) of deprecating globals in favor of using Single section entries for “global” content. So we won’t be adding revision support to globals at this point.
